<launch>
  <node name="ros_miivii_gmsl" pkg="ros_miivii_gmsl" type="ros_miivii_gmsl_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="camCount" value="4" />
    <param name="enable_sync" value="1" />
    <param name="fps" value="25" />
  </node>
  <group ns="ros_miivii_gmsl/image_gmsl_raw0" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
  <group ns="ros_miivii_gmsl/image_gmsl_raw1" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
  <group ns="ros_miivii_gmsl/image_gmsl_raw2" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
  <group ns="ros_miivii_gmsl/image_gmsl_raw3" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>

  <node name="ros_miivii_gmsl1" pkg="ros_miivii_gmsl" type="ros_miivii_gmsl_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="camCount" value="4" />
    <param name="enable_sync" value="1" />
    <param name="fps" value="25" />
  </node>
  <group ns="ros_miivii_gmsl1/image_gmsl_raw0" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
  <group ns="ros_miivii_gmsl1/image_gmsl_raw1" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
  <group ns="ros_miivii_gmsl1/image_gmsl_raw2" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
  <group ns="ros_miivii_gmsl1/image_gmsl_raw3" >
    <rosparam param="disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
    </rosparam>
  </group>
</launch>
